waffle-compiler/src/config/inputToConfig.ts

```diff
@@ -16,8 +16,8 @@ export function inputToConfig(input: InputConfig) {
         deprecate('sourcesPath', 'Use sourceDirectory instead.');
         set('sourceDirectory', input[key]);
       } else if (key === 'targetPath') {
-        deprecate('targetPath', 'Use outputDirectory instead.');
-        set('outputDirectory', input[key]);
+        deprecate('targetPath', 'Use compileOutputDirectory instead.');
```

Comment:
This is not that easy. You cannot just rename the configuration variable like this. Consider what will happen if this gets released.

Somebody had this nice config:
```js
// .waffle.config.json
{
  "outputDirectory": "build"
}
```
This PR changes the configuration to not use `outputDirectory` but `compileOutputDirectory`. Yet not only is this person not notified about this change, his previous configuration stops working.

There is no need to introduce this unnecessary breaking change.

Reply:
I believe the configuration could totally work with the following keys and defaults:
```js
{
  "outputDirectory": "./build",
  "flattenerOutputDirectory": "./build/flat"
}
```